analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Compliant.cs

```diff
@@ -16,4 +16,7 @@ class SupplyParameterFromQueryNeedsRoutableComponent_Compliant : ComponentBase
 
     [SupplyParameterFromQuery]
     public string ParameterAttributeMissing { get; set; } // Compliant: missing [Parameter]
+
+    [Parameter]
+    public TimeSpan TimeSpanParam { get; set; } // Compliant: missing [SupplyParameterFromQuery]
```

Comment:
At this point, we might just change the type of SupplyParameterFromQueryAttributeMissing to TimeSpan and test both rules at the same time